client/components/shipping/packages/style.scss

```diff
@@ -9,31 +13,38 @@
     .package-type {
         width: 10%;
         @include breakpoint( '<660px' ) {
-            width: 12%;
-    }
+            width: 11%;
+        }
 
-    .gridicon {
-        border-radius: 50%;
-        background-color: $gray;
-        fill: white;
-        padding: 4px;
+        .gridicon {
+            border-radius: 50%;
+            background-color: lighten( $gray, 10% );
+            fill: white;
+            padding: 4px;
+            margin-bottom: -4px;
         }
     }
 
     .package-name {
-        width: 55%;
+        width: 51%;
```

Comment:
haha. there's math involved here! i had to take out my calculator and everything.